docs/kyma/04-02-local-installation.md

```diff
@@ -128,59 +103,22 @@ To learn how to test Kyma, see [this](#details-testing-kyma) document.
 Use the `minikube.sh` script to restart the Minikube cluster without reinstalling Kyma. Follow these steps to stop and restart your cluster:
 
 1. Stop the Minikube cluster with Kyma installed. Run:
-  ```
-  minikube stop
-  ```
+   ```
+   minikube stop
+   ```
 2. Restart the cluster without reinstalling Kyma. Run:
-    <div tabs>
-      <details>
-      <summary>
-      MacOS
-      </summary>
-
-      ```
-      ./scripts/minikube.sh --domain "kyma.local" --vm-driver hyperkit
-      ```
-      </details>
-      <details>
-      <summary>
-      Linux
-      </summary>
-
-      ```
-      ./scripts/minikube.sh --domain "kyma.local" --vm-driver virtualbox
-      ```
-      </details>
-    </div>
+   ```bash
+   kyma provision minikube
+   ```
 
-The script discovers that a Minikube cluster is initialized and asks if you want to delete it. Answering `no` causes the script to start the Minikube cluster and restarts all of the previously installed components. Even though this procedure takes some time, it is faster than a clean installation as you don't download all of the required Docker images.
+The Kyma CLI discovers that a Minikube cluster is initialized and asks if you want to delete it. Answering `no` causes the Kyma CLI to start the Minikube cluster and restarts all of the previously installed components. Even though this procedure takes some time, it is faster than a clean installation as you don't download all of the required Docker images.
 
 To verify that the restart is successful, run this command and check if all Pods have the `RUNNING` status:
 
 ```
 kubectl get pods --all-namespaces
 ```
 
-## Enable Horizontal Pod Autoscaler (HPA)
-
-By default, the Horizontal Pod Autoscaler (HPA) is not enabled in your local Kyma installation, so you need to enable it manually.
-
-Kyma uses the `autoscaling/v1` stable version, which supports only CPU autoscaling. Once enabled, HPA automatically scales the number of lambda function Pods based on the observed CPU utilization.
-
->**NOTE:** The `autoscaling/v1` version does not support custom metrics. To use such metrics, you need the `autoscaling/v2beta1` version.
-
-Follow these steps to enable HPA:
-
-1. Enable the metrics server for resource metrics by running the following command:
-  ```
-  minikube addons enable metrics-server
-  ```
-
-2. Verify if the metrics server is active by checking the list of addons:
-  ```
-  minikube addons list
-  ```
-
 ## Troubleshooting
```

Comment:
Please remove the TROUBLESHOOTING section - it's extracted to a separate document now